### PR TITLE
fix(popin): del className bulle-installer

### LINKED ISSUE #1026: List of profile with add on installed : sometime it opens firefox store when I click on follow
**To Reproduce**
1. J'installe prod firefox
2. Je clique sur voir l'exemple d'un profil non suivi
3. Il me propose de suivre le contributeur, je clique sur Suivre.
4. Et là il m'ouvre automatiquement un nouvel onglet du firefox store alors que je l'ai bien installé déjà (et sans passer par la case vous devez installez)
5. Mais si je reviens à mon onglet initial, je suis bien le contributeur et je peux voir l'exemple désormais

**Expected behavior**
A l'étape 4, la page profil devrait juste me confirmer que je suis le contributeur, sans ouvrir de nouvel onglet vers le store.

**Extension**
 - Firefox 3.77.1
 - Probablement aussi 3.76 
 
 Le bug ne se reproduit pas en chrome proding 3.77.1

### PR DESCRIPTION
**Pull request by gregoirelacoste**
closes #1026

### CODE REVIEW

File: src/app/profiles/App/ContextPopin/ContextPopin.tsx [hunk 1]
```diff
@@ -20,10 +20,7 @@ const ContextPopin = ({ setContextPopin, contextPopin }: ContextPopinProps) => {
     <Popin setOpened={() => setContextPopin(initialState)} opened={opened}>
       <PopinParagraph align={'center'}>{content.text}</PopinParagraph>
       {content.btn && (
-        <BackgroundButton
-          className="bulle-installer"
```
Comment (gregoirelacoste) [inline src/app/profiles/App/ContextPopin/ContextPopin.tsx:24]: it's enough to fix the issue ? 
how to test elsewhere prod ?

Comment (gregoirelacoste) [inline src/app/profiles/App/ContextPopin/ContextPopin.tsx:24]: it's enough ?
how to test elsewhere prod ?

**Review by JalilArfaoui — APPROVED** [review]
I’m not fan of the generic approach here, but it seems that should do …

a couple of readings I find interesting :
- https://sandimetz.com/blog/2016/1/20/the-wrong-abstraction
- https://alexkondov.com/the-cost-of-wrong-abstractions/

**Comment by JalilArfaoui** [comment]
@gregoirelacoste Did you forget to merge this ?

**Pull request merged**